frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-settings.ts

```diff
@@ -0,0 +1,29 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+
+export namespace ExtensionProperties {
+  export type ClusterGlobalConfig = {
+    /** Unique identifier for the cluster config resource instance. */
+    uid: string;
```

Comment:
While we currently use `uid` in the static extension, we should update to `id` to be consistent with other extensions where `id` denotes a unique identifier.